Sources/TuistLoader/Models+ManifestMappers/TargetAction+ManifestMapper.swift

```diff
@@ -64,15 +64,7 @@ extension XcodeGraph.TargetScript {
     }
 
     private static func absolutePaths(for paths: [Path], generatorPaths: GeneratorPaths) throws -> [AbsolutePath] {
-        try paths.map { (path: Path) -> [AbsolutePath] in
-            // avoid globbing paths that contain variables
-            if path.pathString.contains("$") {
-                return [try generatorPaths.resolve(path: path)]
-            }
-            let absolutePath = try generatorPaths.resolve(path: path)
-            let base = try AbsolutePath(validating: absolutePath.dirname)
-            return try base.throwingGlob(absolutePath.basename)
-        }.reduce([], +)
+        return try paths.map { try generatorPaths.resolve(path: $0) }
```

Comment:
Globbing here prevents adding missing files to a target script
![Screenshot 2024-08-09 at 18 07 53](https://github.com/user-attachments/assets/8662a37c-09e8-42d4-822b-2b067c8b16c2)